LSPS6/README.md

```diff
@@ -0,0 +1,130 @@
+# LSPS6
+
+| Name    | `.well-known`      |
+| ------- |--------------------|
+| Version | 1                  |
+| Status  | For Implementation |
+
+## Motivation
+
+The purpose is to provide a standard way for clients to obtain extra information from the LSP.
+There might be information that is not available through regular LSPS messages.
+An example for such information could be the pricing strategy for LSPS1.
+Optionally there is also a mechanism to show authoritatively that the information
+is coming from the LSP in order to prevent spoofing.
+
+### Actors
+
+The 'LSP' is the API provider, and acts as the server.
+The 'client' is the API consumer.
+
+## Well-Known URI
+
+[Well-known URIs][] are a way to discover information about a service.
+The LSP SHOULD serve a well-known URI `/.well-known/bitcoin` to allow for public key discovery.
+
+```json
+{
+  "icon": "https://acme.com/icon.png",
+  "logo": "https://acme.com/logo.png",
+  "email": "support@acme.com",
+  "nostr": "npub1w...s2mfp9",
+  "twitter": "@acme",
+  "telegram": "@acme",
+  "lightning": {
+    "mainnet": {
+      "public_keys": [
+        "0326e6...2833a3",
+        "0305f5...2e4e45"
+      ]
+    },
+    "testnet": {
+      "public_keys": [
+        "03f060...869c00"
+      ]
+    },
+    ...
+  }
+}
+```
+
+- `icon` LSP MAY include the URL of its icon.
+- `logo` LSP MAY include the URL of its logo.
+- `email` LSP MAY include its support email.
+- `nostr` LSP MAY include its nost public key.
+- `twitter` LSP MAY include its Twitter handle.
+- `telegram` LSP MAY include its Telegram handle.
+- `lightning`
+  - `mainnet` LSP SHOULD include its mainnet lightning node public keys.
+  - `testnet` LSP MAY include its testnet lightning node public keys.
+  - `...` LSP MAY include its public keys for other network types.
+
+[Well-known URIs]: https://datatracker.ietf.org/doc/html/rfc8615
```

Comment:
I would like to have all data in the `lsps6.get_info` api. Having the data on the web and on the lightning node includes a redundancy that is unnecessary IMO.

Using a web server to provide spoof resistance is great. But I suggest just providing a list of nodes ids in `/.well-known/bitcoin`.

Reply:
removed the socials and redundancy.